proxy/envoy/resources.go

```diff
@@ -113,6 +113,10 @@ const (
 	// ZipkinCollectorEndpoint denotes the REST endpoint where Envoy posts Zipkin spans
 	ZipkinCollectorEndpoint = "/api/v1/spans"
 
+	// ListenersAlpnProtocols denotes the the list of ALPN protocols that the listener
+	// should expose
+	ListenersAlpnProtocols = "h2,http/1.1"
```

Comment:
nit: `ListenersALPNProtocols`
acronyms should be capitalized. ex:
https://golang.org/pkg/net/http/#HandlerFunc.ServeHTTP

Reply:
maybe change this to a `[]string` and use `strings.Join(ListenersAlpnProtocols, ",")` to set `AlpnProtocols`?

Reply:
Sure will do, will add it as a `var` under the `const` section